scripts.py

```diff
@@ -376,3 +394,27 @@ def run(self, uploader=None, cmd_args=None):
             with open(filename, 'w') as f:
                 f.write(unicode(feed))
             self.log.info("OPDS feed saved locally at %s", filename)
+
+        if parsed.search_url and parsed.search_index:
+            search_client = ExternalSearchIndex(parsed.search_url, parsed.search_index)
+            annotator = ContentServerAnnotator()
+
+            search_documents = []
+
+            # It's slow to do these individually, but this won't run very often.
+            for work in works:
+                doc = work.to_search_document()
+                doc["_index"] = search_client.works_index
+                doc["_type"] = search_client.work_document_type
+                doc["opds_entry"] = etree.tostring(AcquisitionFeed.single_entry(self._db, work, annotator))
```

Comment:
Yeah, that's what I'm thinking. We'd need to update the OPDS feeds for the full server. Don't have an entry in front of me, but it might also mean the links would need to be updated if a content source was removed, though isn't the best DataSourced version saved to S3 over its lesser counterparts?